bin/misc-functions.sh

```diff
@@ -83,7 +85,7 @@ install_qa_check() {
 		local EPREFIX= ED=${D}
 	fi
 
-	cd "${ED}" || die "cd failed"
+	cd "${D}" || die "cd failed"
```

Comment:
I wonder if some install QA checks expect to be invoked from `${ED}` rather than `${D}`.

Anyway, this should be safe for the typically (non-prefix) case, so I guess we can roll with it.